.github/actions/security-audit/action.yaml

```
@@ -0,0 +1,212 @@
+      shell: bash
+      if: always() && steps.trivy-check.conclusion == 'failure'
+      run: |
+        python3 ${{ github.action_path }}/scripts/sarif-to-json.py trivy-results.sarif trivy-output.json trivy
+        # Add the contents of the trivy-output.json file to data.json, from which 
+        # notifications about the results of secret checks are generated
+        jq -s '.[0] * .[1]' trivy-output.json data.json | sponge data.json
+        echo reactions="confused" > $GITHUB_ENV
+
+    - name: Run gitleaks check
+      uses: gitleaks/gitleaks-action@v2
+      if: always() && inputs.enable-gitleaks-check == 'true'
+      id: gitleaks-check
+      env:
+        GITHUB_TOKEN: ${{inputs.github-token}}
+        GITLEAKS_LICENSE: ${{inputs.gitleaks-license}}
+        GITLEAKS_ENABLE_COMMENTS: false
+        GITLEAKS_ENABLE_SUMMARY: false
+
+    - name: Add gitleaks check results to data file
+      shell: bash
+      if: always() && inputs.enable-gitleaks-check == 'true'
+      run: |
+        python3 ${{ github.action_path }}/scripts/sarif-to-json.py results.sarif gitleaks-output.json gitleaks
+        # Add the contents of the gitleaks-output.json file to data.json, from which 
+        # notifications about the results of secret checks are generated
+        if [[ -f gitleaks-output.json ]]; then
+          jq -s '.[0] * .[1]' gitleaks-output.json data.json | sponge data.json
+          echo reactions="confused" > $GITHUB_ENV
+        fi
+
+    - name: Create file with scan results
+      shell: bash
+      if: always()
+      run: |
+        if [[ "${{ inputs.enable-vulnerability-checks }}" == "true" ]]; then
+          TEMPLATE="${{ github.action_path }}/templates/pr-comment-vulnerabilities-template.j2"
+        else
+          TEMPLATE="${{ github.action_path }}/templates/pr-comment-secrets-template.j2"
+        fi
+        python3 ${{ github.action_path }}/scripts/generate-message.py \
+          "$TEMPLATE" \
+          data.json \
+          pr-comment.md
+        cat pr-comment.md > $GITHUB_STEP_SUMMARY
+
+    - name: Find PR comment with secret-check results
+      uses: peter-evans/find-comment@v3
+      if: always() && github.event_name == 'pull_request'
+      id: find-secrets-comment
+      with:
+        issue-number: ${{ github.event.number }}
+        comment-author: "github-actions[bot]"
+        body-includes: Summary of the secrets check
+
+    - name: Find PR comment with vulnerabilities-check results
+      uses: peter-evans/find-comment@v3
+      if: always() && github.event_name == 'pull_request'
+      id: find-vulnerabilities-comment
+      with:
+        issue-number: ${{ github.event.number }}
+        comment-author: "github-actions[bot]"
+        body-includes: Summary of the vulnerabilities check
+
+    - name: Upload secrets-checks results as a PR comment
+      if: always() && github.event_name == 'pull_request' && inputs.enable-pr-comment == 'true' && steps.find-secrets-comment.outputs.comment-id != ''
```

Is it possible to do just like this here?
```suggestion
      if: always() && inputs.enable-pr-comment == 'true' && steps.find-secrets-comment.outputs.comment-id != ''
```